connector/mqtt/vernemq/v2k-brigde/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "v2kbridge",
+  "version": "1.0.0",
+  "description": "service to bridge verne to kafka",
```

Comment:
Service to bridge messages from VerneMQ to Apache Kafka according to dojot rules.